src/Swarm/Game/Entity.hs

```diff
@@ -345,6 +388,7 @@ instance FromJSON Entity where
               <*> (v .: "description")
               <*> v .:? "orientation"
               <*> v .:? "growth"
+              <*> v .:? "combustion"
```

Comment:
Can we also update `data/schema/entities.json` and `scenarios/README.md` to match?  I know we haven't been keeping it completely up-to-date but it would be nice to get it back in shape.

Reply:
Updated.  Though it was tedious to reproduce the info in `scenarios/README.md`.  Do we have a way to auto-generate that from `data/schema/entities.json`?

Reply:
Agreed.  No, we don't, but that would be nice.

Reply:
> it was tedious to reproduce the info in scenarios/README.md. Do we have a way to auto-generate that from data/schema/entities.json?

For posterity: fixed in #1441